lang/go.md

```diff
@@ -0,0 +1,99 @@
+---
+lang: Go
+syntax: go
+title: Introduction to the Go Programming Language | Go Explained
+author: krishguptadev
+---
+
+```go
+package main
+
+import "fmt"
+
+func main() {
+    fmt.Println("hello world")
+}
+```
+
+## 📚 What is go?
+
+go is a statically typed, compiled programming language with a similar syntax to C but with garbage collection, memory safety, CSP-style  concurrency and structural typing. It is a tool for building simple and reliable software, and it is used by Google, Facebook, Instagram, Netflix, Pinterest, and many other companies. 
+
+## 🤔 Why to use Go?
+
+Go is a great language for building software. It is a statically typed language, which means you can't mix types in the same program. It is a compiled language, which means it can run faster than other languages. It is an object oriented language, which means you can build programs that are easy to read and maintain, and it is easy to extend.
```

Comment:
```suggestion
Go is a great language for building software. It is a statically typed language, which means you can't mix types in the same program. It is a compiled language, which means it can run faster than other languages. It is an object-oriented language, which means you can build programs that are easy to read and maintain, and it is easy to extend.
```